src/main/resources/assets/chisel/lang/de_de.lang

```diff
@@ -0,0 +1,2774 @@
+item.chisel.chisel_iron.name=Meißel
+item.chisel.chisel_diamond.name=Diamantmeißel
+item.chisel.chisel_hitech.name=iMeißel
+item.chisel.offsettool.name=Ender-Versatzstab
+
+chisel.spawnwither=Ein Meißel kann etwas gefährliches sein...
```

Comment:
Remember you do not always have to translate literally. If something makes more sense another way, or just sounds better, you're free to do it differently as long as the overall meaning stays the same.

If you prefer to keep your translation, "Gefährliches" will need to be capitalized.